packages/datadog-plugin-bluebird/test/index.spec.js

```diff
@@ -1,5 +1,10 @@
 'use strict'
 
 const assertPromise = require('../../dd-trace/test/plugins/promise')
+const semver = require('semver')
 
 assertPromise('bluebird')
+
+assertPromise('bluebird', bluebird => {
+  return bluebird.getNewLibraryCopy()
+}, version => { semver.intersects(version, '^2.11.0 || ^3.4.1') })
```

Comment:
Can the function be moved to `assetPromise` and pass only the range instead?

Reply:
yup, updated, moved the semver require and versionrange filtering function to `assertPromise` and exposed versionRange as an option to pass in. Should be a bit more DRY now, and i actually ended up needing it for promise-js updates so makes sense move to promise specs